.github/workflows/easy-cp.yml

```diff
@@ -47,29 +47,26 @@ jobs:
           git config user.email "<>"
           git fetch origin $RELEASE_BRANCH
           git fetch origin master
-          git checkout -b $RELEASE_BRANCH --track origin/$RELEASE_BRANCH
+          git checkout -b cp-${CHANNEL}-${COMMIT_SHA} --track origin/$RELEASE_BRANCH
           git cherry-pick $COMMIT_SHA
       # TODO(xilaizhang): remove this step once the template is available on release branches.
       - name: Get CP Template
         run: |
           curl -o PULL_REQUEST_CP_TEMPLATE.md https://raw.githubusercontent.com/flutter/flutter/master/.github/PR_TEMPLATE/PULL_REQUEST_CP_TEMPLATE.md
       - name: Create PR on CP success
         if: ${{ steps.attempt-cp.conclusion == 'success' }}
-        uses: peter-evans/create-pull-request@153407881ec5c347639a548ade7d8ad1d6740e38
-        with:
-          base: ${{ env.RELEASE_BRANCH }}
-          branch: cp-${{ env.CHANNEL }}-${{ env.COMMIT_SHA }}
-          path: flutter
-          token: ${{ secrets.FLUTTERACTIONSBOT_CP_TOKEN }}
-          labels: |
-            cp: review
-          title: '[${{ env.CHANNEL }}-cherrypick] cherrypicks commit ${{ env.COMMIT_SHA }} from PR ${{ github.event.pull_request.title }}'
-          body-path: PULL_REQUEST_CP_TEMPLATE.md
+        working-directory: ./flutter
+        run: |
+          git push -u origin cp-${CHANNEL}-${COMMIT_SHA}
+          gh pr create --title "[cp-${CHANNEL}]${PR_TITLE}" --body-file ../PULL_REQUEST_CP_TEMPLATE.md --base ${RELEASE_BRANCH} --label "cp: review"
```

Comment:
Should we be explicitly using the CP_TEMPLATE.md from HEAD instead of what's on the branch?

Reply:
Right currently the PULL_REQUEST.md is still a downloaded file from HEAD instead of a file on the branch.

There is a step before this step that downloads the template from HEAD. Couldn't remove the download logic yet because this file doesn't exist on release branch (e.g. 3.16-candidate.0). If we remove the download step, then when we checkout the release branch, this template becomes unaccessible.

Reply:
In that case, should the git push be to the action bot's fork?

Reply:
Umm I thought in design doc we [agreed on creating the branch on the flutter/flutter repo directly and push, removing the usage of fork](https://docs.google.com/document/d/1-KZyfJ7dA6fGM33nvPaXfpH02b72LjHZ3fuIj6WUkzM/edit?disco=AAAA_fD49uM).

To push to action bot's fork, we need a separate workflow to keep the action bot's fork always up to date with upstream.

(Umm i think Casey previously mentioned that creating branches on flutter/flutter are cheap because they are all refs)

Reply:
You don't need to maintain the fork's branch. See my Cocoon branch at https://github.com/CaseyHillers/cocoon, hasn't been updated in 4 years :-)

Instead, make sure to run `git fetch` on the upstream's RC before creating a tracking branch on the fork.

Reply:
umm I can try reverting back to the fork method.

I thought in previous discussions, it was pointed out creating branches on upstream directly is cheaper and easier, and therefore we switched to the current method of creating branches on upstream. What would be the verdict on why we are now deciding to revert back to the fork method?

Reply:
updated PR to use fork.

Tested: https://github.com/flutter/flutter/pull/140530 which is opened from fork to upstream. [sample workflow](https://github.com/XilaiZhang/miscellaneous-side-project/actions/runs/7294792678/job/19880230894)